# SERVICE TYPE: GetGeofence
# ------------------------------------------------------------------------------
# This service return a geofence list (lat, lon, alt) stored in the geofencing node

---
bool success                    # whether the geofence has been set or not
geofencing/Geofence[] geofences   # geofences stored in memory
# ------------------------------------------------------------------------------
